swan-lake/learn/using-the-cli-tools.md

```diff
@@ -167,7 +167,7 @@ These commands allow you to work with the Ballerina Central to share Ballerina m
 </tr>
 <tr>
 <td class="cCommand">push</td>
-<td class="cDescription">Upload a module to Ballerina Central. For more information, see <a href="/swan-lake/learn/publishing-modules-to-ballerina-central">Publishing Modules to Ballerina Central</a>.
+<td class="cDescription">Upload a module to Ballerina Central. For more information, see <a href="/swan-lake/learn/publishing-packages-to-ballerina-central">Publishing Modules to Ballerina Central</a>.
```

Comment:
```suggestion
<td class="cDescription">Upload a package to Ballerina Central. For more information, see <a href="/swan-lake/learn/publishing-packages-to-ballerina-central">Publishing Modules to Ballerina Central</a>.
```

Reply:
@azinneera changes in `using-the-cli-tools.md` will be addressed in [this PR](https://github.com/ballerina-platform/ballerina-dev-website/pull/1705). I just updated the reference.